README.md

```diff
@@ -347,6 +347,20 @@ To avoid storage charges for retaining the temporary composite objects, consider
 ### Google Cloud Storage Class
 Due to the quick creation and deletion of composite objects, we recommend that only the [Standard storage class](https://cloud.google.com/storage/docs/storage-classes) is applied to your bucket to minimize cost and maximize performance.
 
+### Throughput and QPS Limits
+Many machine learning efforts opt for a highly distributed training model leveraging tools such as Pytorch Lightning and Ray. These models are compatible with Dataflux, but can often trigger the rate limits of Google Cloud Storage. This typically manifest in a 429 error, or slower than expected speeds while running distributed operations. Details on specific quotas and limits can be found [here](https://cloud.google.com/storage/quotas).
+
+#### Egress Throughput Limits
+429 errors acompanied with messages indicating `This workload is drawing too much egress bandwidth from Google Cloud Storage` or `triggered the Cloud Storage Egress Bandwidth Cap` indicate that the data throughput rate of your workload is exceeding the maximum capacity of your Google Cloud Project. To address these issues, you can take the following steps:
+
+1. Ensure that object composition is [disabled](#composite-objects) for dataflux download operations.  
```

Comment:
Could you add a bullet point explaining this recommendation? In fact I'm not sure it makes sense here - if you're hitting egress throughput limits, then you might still hit them if you disable compose, but maybe not.

Reply:
You could definitely still hit them, however they are much more retry-able if its on a regular download. Due to the extra load caused by compose, the backoff tends to progress much further when trying to compose before it can resume.

Reply:
Maybe this is in the wrong section then, won't that give a QPS error rather than an egress throughput error?

Reply:
I put it as a low priority troubleshooting step in this segment since it could help, but I put it as the primary troubleshooting step for the QPS error use case listed below if that seems ok. I can fully remove it if thats preferred tho.